src/main/java/com/rtb/projectmanagementtool/task/Status.java

```diff
@@ -0,0 +1,7 @@
+package com.rtb.projectmanagementtool.task;
+
+/** Enum containing status options for a task. */
+public enum Status {
+  COMPLETE,
+  INCOMPLETE
+}
```

Comment:
Instead of creating a separate file for this, would this have worked as a public member of TaskData?

```
class TaskData {
  public enum Status {
    ...
  }
}
```

Reply:
I tried, but it only works when it's in its own file